.github/workflows/format-code.yml

```diff
@@ -0,0 +1,47 @@
+name: "Format C code"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  format:
+    name: "Format C code"
+    runs-on: ubuntu-18.04
```

Comment:
Should we be able to change to `ubuntu-latest` now? 
Not sure if there is a concern for compatibility with local, or if we are able to "ignore" local once this is in.
If  we can change it, it should probably change in the `build.yml` also

Reply:
Especially for the formatting code here, I think we should keep this pinned to a specific version so that we aren't surprised by changes to indent. It's a small thing but may avoid confusion in the future.

ubuntu-latest is still ubuntu-18.04 AFAICT